resources/content/articles/cae-information.html

```diff
@@ -0,0 +1,43 @@
+---
+title: Certificate in Advanced English (CAE)
+link: cae-information
+category: Englisch
+thumbnail: cae_information_thumbnail
+thumbnail_slide: cae_information_slide
+---
+<h3>Certificate in Advanced English (CAE)</h3>
+
+<figure class="float-left" style="width:30%">
+    <v-image name="cae_information_banner" alt="Your key to success">
+</figure>
+
+<p>
+    Die CAE Prüfung ist eine international anerkannte Sprachprüfung,
+    durch die der für die Aufnahme eines Studiums in Großbritannien oder
+    den USA benötigte Sprachnachweis für eine Vielzahl von Universitäten
+    und universitären Einrichtungen erbracht werden kann. Das nach
+    Bestehen der Prüfung ausgestellte Zertifikat weist bei Bewerbungen im
+    In- und Ausland Kenntnisse und Fertigkeiten in der englischen Sprache
+    auf dem Niveau C1 des Gemeinsamen europäischen Referenzrahmens für das
+    Sprachenlernen nach.
+</p>
+
+<p>
+    Die CAE Prüfung erstreckt sich über die Bereiche: Reading and Use of
+    English, Writing, Listening, Speaking.
+</p>
+<p>
+    Schülerinnen und Schüler, die über sehr gute bis gute englische Sprachkenntnisse verfügen, können sich über das
+    Willibald-Gluck-Gymnasium für diese externe Sprachprüfung anmelden. Die ermäßigte Teilnahmegebühr für das CAE
+    beträgt für Schüler 157 €. Die Anmeldung ist im Herbst möglich, die Schülerinnen und Schüler der Q11 und Q12 werden
+    rechtzeitig von ihren Kursleitern informiert. Die Prüfung selbst findet jeweils im März statt.
+</p>
+
+<p class="fett" style="margin-bottom: 1em;">
+    <span>Termine:</span>
```

Comment:
Also:
1. Why is this in a `span`?
2. Why is this even here, if there aren't any? If it is to signal that there aren't any, it probably should say something about that.